README.md

```diff
@@ -71,8 +71,16 @@ Command Name | Command Description
 `cf backup BACKUP_ID` | Show the information about a particular backup.
 ` cf list-backup ` | Show the list of all backups present in the space.
 ` cf list-backup SERVICE_INSTANCE_NAME ` | Show the list of all backups for the given service-fabrik service instance.
+` cf list-backup --guid SERVICE_INSTANCE_GUID` | Show the list of all backups for the given service-fabrik service instance. The argument has to be the guid of the service instance. (Works even for a deleted instance.)
+`cf list-backup SERVICE_INSTANCE_NAME --deleted` | Shows the list of all backups for a deleted service-fabrik service instance. (Works only for a deleted service-instance.)
+`cf instance-events` | Lists all events including create, update and delete events triggered for all service instances present in the space.
```

Comment:
"List all create, update and delete service instance events in the space"

Reply:
Done.

Reply:
Done.